bin/qunitjs

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env node
+/* eslint-env node */
+
+var path = require( "path" );
+var program = require( "commander" );
+var fs = require( "fs-extra" );
+var walkSync = require( "walk-sync" );
+var run = require( "./run" );
+
+var pkg = fs.readJsonSync( path.resolve( __dirname, "..", "package.json" ) );
```

Comment:
would `require( '../package.json' );` work here? I believe require, unlike fs, is always scoped to where the code is authored, so it shouldn't need `__dirname`.